src/components/look-controls.js

```diff
@@ -20,6 +20,15 @@ module.exports.Component = registerComponent('look-controls', {
     this.setupMouseControls();
     this.setupHMDControls();
     this.bindMethods();
+
+    // Enable grab cursor class on canvas.
+    var sceneEl = this.el.sceneEl;
```

Comment:
True, but just because that is the way it is now doesn't mean we couldn't improve it. Even in the case of `sceneEl`, I question the convention, there is still my problem of declaring variables up top and not knowing where/how they are used. And if I delete the block around the area where it *is* used, I have to scroll up and delete the declaration.